Messaging.md

```diff
@@ -377,12 +402,13 @@ See [the Oracle specifications](./Oracle.md#oracle-event) for more details.
 ### The `oracle_announcement` Type
 
 This type contains an `oracle_event` and a signature certifying its origination.
+As oracle announcements can be broadcast directly, they are encoded as [wire messages](#wire-messages).
 See [the Oracle specifications](./Oracle.md#oracle-announcements) for more details.
 
-#### Version 0 `oracle_announcement`
+#### `oracle_announcement`
 
-1. type: 55332 (`oracle_announcement`)
-2. data:
+1. type: 42786
```

Comment:
why are these changed? This seems to break backwards compatability for no good reason?

Reply:
Mainly just to make the range of types we use smaller. I don't mind reverting this, I assume you would prefer?

Reply:
Yes please. It's such a critical data structure in existing deployed applications such as our oracle explorer and Atomic Finance's app. I don't have any problems with restricting the range of types with future announcements we define in the spec.

Reply:
Updated!